CONTRIBUTING.md

```diff
@@ -22,11 +22,20 @@ If you'd like to request a feature you deem necessary and useful, open a new Git
 
 If you wish to contribute to the project by submitting code directly, please first leave a comment under the relevant issue or file a new issue, describe the changes you are about to make.
 
-To avoid multiple pull requests resolving the same issue, let others know you are working on it by saying so in a comment, or ask the issue to be assigned to yourself.
+As per our project's guidelines, we adhere to [conventional commits][conv-commits]. Therefore, it's expected that all PRs align with this convention. Should a PR not meet these standards, we'll kindly request a review and revision.
+
+Please use the provided Pull Request templates according to your needs. Currently, there are two to choose from:
+- [General PR](.github/PULL_REQUEST_TEMPLATE/pull_request_template.md): for all other PRs, including bug fixes, adding features etc.
```

Comment:
It looks like the link for General PR still isn't working, looking at the files inside `PULL_REQUESR_TEMPLATE` folder, i see it's called `general_template.md` instead of `pull_request_template.md`.

Reply:
🤦🏽 my bad